CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+# Before you contribute to the SPDX License List
+Please familiarize yourself with the SPDX License List and its supporting documentation, so that you understand the pertinent context around the list itself:
+* [Overview of the SPDX License List](https://spdx.org/spdx-license-list/license-list-overview) explains some key background, principles for the inclusion of a license on the SPDX License List, as well as an explanation of the fields contained on the list.
+* [SPDX License List Matching Guidelines](https://spdx.org/spdx-license-list/matching-guidelines) provides guidelines to be used for the purposes of matching licenses and license exceptions against those included on the SPDX License List. 
+
+# How to contribute/participate: general
+The SPDX License List is maintained by the SPDX Legal Team. Work for the SPDX Legal Team is done both via the mailing list and bi-weekly calls. In particular, the bi-weekly calls are used to discuss topics and issues that may be difficult to only discuss via email.
+You can join the mailing list and otherwise manage your subscription [via spdx-legal mailman](https://lists.spdx.org/mailman/listinfo/spdx-legal). Information on the bi-weekly calls and the working area for the SPDX Legal Team, can be found on the [SPDX wiki](https://wiki.spdx.org/view/Legal_Team)
+
+## Request a new license or exception be added to the SPDX License List
+The SPDX Legal Team greatly appreciates proposals for new open source licenses or exceptions to be added to the SPDX License List.  To request that a license or exception be added to the SPDX License List, please:
+
+1)  First, review the [license list overview and inclusion principles](https://spdx.org/spdx-license-list/license-list-overview).
+
+2)  Check the historical license and exceptions [tracking page](https://docs.google.com/spreadsheets/d/11AKxLBoN_VXM32OmDTk2hKeYExKzsnPjAVM7rLstQ8s/edit?pli=1#gid=695212681) and Issues with label ADD to ensure this license or exception has not been previously requested. 
+
+3) Submit your request with all of the following information either via a new Issue or via the SPDX-legal mailing list
+If you are not on the SPDX-legal mailing list, please join here. If you send your license request without joining the mailing list, this will greatly slow down correspondance in both directions.
+
+* Issue name or email subject line: "New License/Exception Request: <license/exception name>" 
```

Comment:
I'd [rather][1] see an issue template for this information.  Then we can drop it from `CONTRIBUTING.md`, and folks creating new issues get the template in front of them automatically.

[1]: https://github.com/spdx/license-list-XML/pull/538#commitcomment-26393643

Reply:
I like the idea of an issue template and @bradleeedmondson has suggested this as well. Found some info here: https://github.com/blog/2111-issue-and-pull-request-templates 
 @goneall - would having this template in the root directory mess anything up? If not, @wking could you have a go at the template?